garageScript/c0d3-app
# Add the ability to unlink discord from user profile page

Closes #2243 

Merging this PR will add the ability for the user link/unlink their discord account from the profile page.

To test it:

1. Go to `/profile/YOUR_USERNAME or admin`
2. Click `unlink` discord if you already connected your account or `connect to discord`
3. Confirm the action from the modal

Once the user unlink their account, the page reloads to load the new user data from the db (e.g., removing their profile picture).
